omero/sysadmins/unix/install-web-trial.txt

```diff
@@ -0,0 +1,348 @@
+OMERO.web WSGI deployment (TRIAL)
+=================================
+
+OMERO.web is the web application component of the OMERO platform which
+allows for the management, visualization (in a fully multi-dimensional
+image viewer) and annotation of images from a web browser. It also
+includes the ability to manage users and groups.
+
+Deploying OMERO.web separately to OMERO.server is recommended as they
+perform best under different circumstances and require a different set of
+dependencies. This guide uses the example of deploying with nginx and
+gunicorn.
+
+Prerequisites
+-------------
+
+-  Python 2.7
+
+   -  :pypi:`virtualenv` (optional) tool to create isolated Python environments
+
+   -  :pypi:`PyPI <>` a package management system used to install and manage
+      software packages written in Python. PyPI is already installed if
+      you are using Python 2 >=2.7.9
+
+   -  `omego <https://github.com/ome/omego>`_ OME package management system
+
+   -  `ZeroC`_ IcePy 3.6 (IcePy 3.5 is also supported although requires global
+      installation from RPM)
+
+   -  `Pillow`_ <3.4
+
+   -  `NumPy <http://www.numpy.org>`_ >=1.9 
+
+-  A `WSGI <http://wsgi.readthedocs.org>`_ capable web server such as
+   `nginx <http://nginx.org/>`_ and `gunicorn <http://docs.gunicorn.org/>`_,
+
+
+Gunicorn default configuration
+------------------------------
+
+Install above OMERO.web dependences using the package management tools:
+
+Virtualenv is an optional step:
+
+::
+
+   $ virtualenv omeroweb
+   $ source omeroweb/bin/activate
+
+
+::
+
+   $ pip install omego
+   $ omego download py
+   2016-08-24 16:20:14,252 [omego.artifa] INFO  Checking http://downloads.openmicroscopy.org/omero/5.2.5/./artifacts/OMERO.py-5.3.0-ice36-b29.zip
+   2016-08-24 16:20:14,252 [omego.fileut] INFO  Downloading http://downloads.openmicroscopy.org/omero/5.2.5/./artifacts/OMERO.py-5.3.0-ice36-b29.zip
+   2016-08-24 16:20:14,514 [omego.artifa] INFO  Unzipping OMERO.py-5.3.0-ice36-b29.zip
+  $ pip install -r OMERO.py/web/requirements-py27-experimental.txt
+  $ ln -s `pwd`/OMERO.py-5.2.5-ice36-b28 `pwd`/OMERO.py
+
+
+.. note:: For more details refer to
+      :djangodoc:`how to install Django 1.8 <topics/install/#install-the-django-code>`
+      or :djangodoc:`upgrade Django to 1.8 <topics/install/#remove-any-old-versions-of-django>`.
+
+
+Additional settings can be configured by changing the following properties:
+
+- :property:`omero.web.application_server.max_requests` to 500
+
+- :property:`omero.web.wsgi_workers` to (2 x NUM_CORES) + 1
+
+  .. note::
+      **Do not** scale the number of workers to the number of clients
```

Comment:
With the bold "do not" I wonder if you want to consider making this a warning rather than a note.